tests/foreman/cli/test_contentviews.py

```diff
@@ -86,20 +93,13 @@ def create_rhel_content(self):
         })
 
         RepositorySet.enable({
-            'name': (
-                'Red Hat Enterprise Virtualization Agents '
-                'for RHEL 6 Workstation (RPMs)'
-            ),
+            'name': REPOSET['rhva6'],
             'organization-id': ContentViewTestCase.rhel_content_org['id'],
-            'product': 'Red Hat Enterprise Linux Workstation',
-            'releasever': '6Workstation',
+            'product': PRDS['rhel'],
+            'releasever': '6Server',
             'basearch': 'x86_64',
         })
-        ContentViewTestCase.rhel_repo_name = (
-            'Red Hat Enterprise Virtualization Agents '
-            'for RHEL 6 Workstation '
-            'RPMs x86_64 6Workstation'
-        )
+        ContentViewTestCase.rhel_repo_name = REPOS['rhva6']['name']
```

Comment:
For reviewers, the important change is that we don't want to use `6Workstation` anymore.

```python
REPOS = {
    'rhva6': {
        'name': (
            'Red Hat Enterprise Virtualization Agents for RHEL 6 Server RPMs '
            'x86_64 6Server'
        ),
    },
}
```